ReactiveUI.XamForms/XamForms/ActivationForViewFetcher.cs

```diff
@@ -17,6 +14,7 @@ public int GetAffinityForView(Type view)
         {
             return
                 (typeof(Page).GetTypeInfo().IsAssignableFrom(view.GetTypeInfo())) ||
+                (typeof(View).GetTypeInfo().IsAssignableFrom(view.GetTypeInfo())) ||
```

Comment:
Hmmm, I think I'll split into distinct methods for now. I don't really want to have one platform differ by having multiple activator implementations.